orga/translations/en.json

```
@@ -297,6 +297,10 @@
       "owner": {
         "label": "Campaign owner",
         "placeholder": "Owner’s first and last name",
+        "search-placeholder": "TODO",
+        "clear-label": "TODO",
+        "expand-label": "TODO",
+        "collapse-label": "TODO",
```

En attendant les bonnes traductions, en voici 😊 
```suggestion
        "search-placeholder": "Search an owner",
        "clear-label": "Clear owner selection",
        "expand-label": "Expand dropdown",
        "collapse-label": "Collapse dropdown",
```